### DISCUSSION TITLE
enforce one space between array elements

### DISCUSSION THREAD

**Discussion by ianzone** [rome/tools#4341] [Suggestions]
Current behavior: 
![image](https://user-images.githubusercontent.com/20043230/229342508-a0b5594d-8978-4464-b43b-b59efc0f1da8.png)

Expected: 
![image](https://user-images.githubusercontent.com/20043230/229342556-05aa2d96-845a-4c96-a1d9-1c8612fa2662.png)

**Comment by ematipico** [comment]
Decorators are not currently supported in Rome:

https://docs.rome.tools/internals/language_support/#typescript-support

**Comment by GabenGar** [comment]
Decorators are [stage 3](https://github.com/tc39/proposal-decorators) already.
But in regards to typescript it's a lot more complex, since at this point it has to support [two types of them](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-0.html#differences-with-experimental-legacy-decorators). I bet it's going to be fun to separate the two for the 3rd-party tools.
